tests/unit/scanner/test_validate_rules_db.py

```diff
@@ -460,143 +462,3 @@ def test_validate_rules_with_valid_rules(
     validate_rules(yaml.safe_load(rules_db_yaml), registry_config)
     out, _ = capfd.readouterr()
     assert out == ""
-
-
-def test_main_with_valid_rules(
-    cli_runner: CliRunner,
-    registry_path: Path,
-    scanners_path: Path,
-    rules_realm_path: Path,
-) -> None:
-    """Test main with valid rules."""
-    _create_module_rules(scanners_path, "namespace/module-name", VALID_RULES_DB_STRING)
-    _create_module_rules(
-        rules_realm_path, "namespace/realm-name", VALID_RULES_DB_STRING
-    )
-    result = cli_runner.invoke(
-        app,
-        [
-            "--registry-path",
-            str(registry_path),
-        ],
-    )
-    assert result.stdout == (
-        "Validating namespace/module-name/rules.yaml\n"
-        "Validating namespace/realm-name/rules.yaml\n"
-    )
-
-
-def test_main_with_valid_imports(
-    cli_runner: CliRunner,
-    registry_path: Path,
-    scanners_path: Path,
-) -> None:
-    """Test main with valid imported rules."""
-    _create_module_rules(
-        scanners_path,
-        "testing-ns/testing-module",
-        VALID_RULES_DB_STRING_WITH_ONLY_IMPORT,
-    )
-    _create_module_rules(scanners_path, "namespace/module-a", VALID_RULES_DB_STRING)
-    result = cli_runner.invoke(
-        app,
-        [
-            "--registry-path",
-            str(registry_path),
-        ],
-    )
-    assert "Validating namespace/module-a/rules.yaml\n" in result.stdout
-    assert "Validating testing-ns/testing-module/rules.yaml\n" in result.stdout
-
-
-def test_main_with_valid_imports_from_realm(
```

Comment:
Is this test not valid anymore?

Reply:
It is, but it's a duplicate of the previous test where we already test validation of rules-realm.